### ISSUE TITLE
Cart form validation hidden

### ISSUE THREAD

**Issue by sanpii** [Sylius/Sylius#9520]
**Sylius version affected**: 1.1.7

**Description**  
I have override the quantity form type to add a minimun quantity validation.

The validation works fine on the product page, but on the cart page, when I update an item with an invalid quantity the symfony debug toolbar display the form error but it’s not display on the page.

**Steps to reproduce**  
```php
<?php

declare(strict_types=1);

namespace AppBundle\Form\Extension;

use Sylius\Bundle\OrderBundle\Form\Type\CartItemType;
use Symfony\Component\Form\AbstractTypeExtension;
use Symfony\Component\Form\Extension\Core\Type\IntegerType;
use Symfony\Component\Form\FormBuilderInterface;
use Symfony\Component\Validator\Constraints\Range;

final class CartItemExtension extends AbstractTypeExtension
{
    public function buildForm(FormBuilderInterface $builder, array $options): void
    {
        $min = 10;
        $builder
            ->add('quantity', IntegerType::class, [
                'attr' => ['min' => $min],
                'data' => $min,
                'label' => 'sylius.ui.quantity',
                'constraints' => [
                    new Range([
                        'min' => $min,
                        'groups' => ['sylius'],
                    ]),
                ],
            ]);
    }

    public function getExtendedType(): string
    {
        return CartItemType::class;
    }
}
```
```yaml
services:
    app.form.extension.type.cart_item:
        class: AppBundle\Form\Extension\CartItemExtension
        tags:
            - { name: form.type_extension, extended_type: Sylius\Bundle\OrderBundle\Form\Type\CartItemType, priority: -1 }
```

**Comment by Prometee** [comment]
@CoderMaggie It's steal a bug, the quantity field will display the validation error on a product page, but when changing it in the cart summary page it won't be displayed.
What it's also awkward is the cart summary will display the updated prices even if the cart form has errors.

What can be done first is to display the error under each quantity field in the cart page :
https://github.com/Sylius/Sylius/blob/master/src/Sylius/Bundle/ShopBundle/Resources/views/Cart/Summary/_item.html.twig#L19
```twig
<span class="sylius-quantity-error ui form">{{ form_errors(form.quantity) }}</span>
```
But it won't avoid getting the cart updated.

**Comment by Prometee** [comment]
@sanpii I'm sure you figured out since then.
From my side I ended up creating the `Constraint` I need with a `OnlyOneCustomProductOrderItemValidator` which is doing exactly what you needed with `$min = 1;`.
Then I decorated `sylius.order_item_quantity_modifier.limiting` to create a new `LimitingOrderItemQuantityModifier` service which set quantity to 1 when the custom product quantity will change.
The only thing that bothers me is that the customer will not be informed that his change has not been taken into account when he change the quantity in the cart summary page.

I'm also thinking to remove the quantity field to a simple text because my custom product need to be bought one by one.
